backend/routes/pages.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+const User = require('../core/user');
+const router = express.Router();
+
+
+const user = new User();
+
+
+router.get('/', (req, res, next) => {
+    let user = req.session.user;
+    
+    if(user) {
+        res.redirect('/home');
+        return;
+    }
+    
+    res.render('index');
+})
+
+
+router.get('/home', (req, res, next) => {
+    let user = req.session.user;
+
+    if(user) {
+        res.render('home', {opp:req.session.opp, name:user.username});
```

Comment:
- Is a render necessary here? I believe you could just respond with JSON here.
- I cannot find any view named "home" in you code, so I don't believe this render will work at all.